### PR TITLE
Fix bug #49 - Submit button when disabled is styled as enabled

### PR DESCRIPTION
**Pull request by vasylmrnchk**

### CODE REVIEW

File: frontend/src/app/modules/auth/register-form/register-form.component.scss [hunk 1]
```diff
@@ -56,6 +56,9 @@
         }
         button:disabled {
             cursor: default;
+            background: #e0d8d7;
+            color: #989898;
+            border: 1px solid #e0d8d7 !important;
```
Comment (Anatolii-Ivanov) [inline frontend/src/app/modules/auth/register-form/register-form.component.scss]: Can't we deal without !important?

**Review by Anatolii-Ivanov — COMMENTED** [review]
Please, fix github actions comments, I know it's not your changes

File: frontend/src/app/modules/auth/register-form/register-form.component.scss (hunk 1 above)
Comment (vasylmrnchk) [reply frontend/src/app/modules/auth/register-form/register-form.component.scss]: > Please, fix github actions comments, I know it's not your changes
Ok

Comment (vasylmrnchk) [reply frontend/src/app/modules/auth/register-form/register-form.component.scss]: > Can't we deal without !important?

I cheched, it's work without !important

File: frontend/src/app/modules/auth/register-form/register-form.component.scss [hunk 2]
```diff
@@ -56,6 +56,9 @@
         }
         button:disabled {
             cursor: default;
+            background: #e0d8d7;
```
Comment (Jyuart) [inline frontend/src/app/modules/auth/register-form/register-form.component.scss:59]: For me, it still looks like enabled
image

Comment (vasylmrnchk) [reply frontend/src/app/modules/auth/register-form/register-form.component.scss:59]: I have testeted in Firefox, Chrome, Opera, and it's works for me with this test data. 
chrome
opera
firefox

**Review by Jyuart — APPROVED** [review]

**Review by aydevelop — APPROVED** [review]
It works great

**Pull request merged**